.coderabbit.yml

```
@@ -0,0 +1,35 @@
+language : "ko-KR"
+
+reviews :
```

YAMLlint (1.37.1)

[error] too many spaces before colon

(colons)